x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/exit_code_draggable.tsx

```diff
@@ -15,12 +15,13 @@ interface Props {
   contextId: string;
   endgameExitCode: string | null | undefined;
   eventId: string;
+  processExitCode: number | null | undefined;
   text: string | null | undefined;
 }
 
 export const ExitCodeDraggable = React.memo<Props>(
-  ({ contextId, endgameExitCode, eventId, text }) => {
-    if (isNillEmptyOrNotFinite(endgameExitCode)) {
+  ({ contextId, endgameExitCode, eventId, processExitCode, text }) => {
+    if (isNillEmptyOrNotFinite(processExitCode) && isNillEmptyOrNotFinite(endgameExitCode)) {
```

Comment:
I am wondering if we really want an AND condition here, I feel like a OR seems safer here. I might missing something too.